app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :email, format: URI::MailTo::EMAIL_REGEXP
```

Comment:
Instead of adding this line, can we change the configuration of devise to do this in `config/devise.rb`

Reply:
Instructive!! Thanks